docs/technical_reference/flowsheets/lsrro.rst

```diff
@@ -0,0 +1,192 @@
+
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Equality constraint between A and B", ":math:`B_{\text{comp}} = AB_{\text{trade-off}} \cdot (A_{\text{comp}})^{3}`"
+
+If ``AB_tradeoff`` is set to ``inequality_constraint``:
+
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Inequality constraint between A and B", ":math:`B_{\text{comp}} \geq AB_{\text{trade-off}} \cdot (A_{\text{comp}})^{3}`"
+
+If ``has_NaCl_solubility_limit`` is set to ``True``:
+
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "NaCl solubility limit", ":math:`X_{\text{Liq, NaCl}} \leq 0.2614`"
+
+If ``permeate_quality_limit`` is not None:
+
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Permeate quality limit", ":math:`X_{\text{Liq, NaCl}_{\text{permeate}}} \leq \text{permeate quality limit}`"
+
+
+
+Degrees of Freedom
+------------------
+
+In addition to the number of stages and water mass recovery of the system, the following variables need to be specified based on the default settings:
+
+   * feed water conditions (flow, temperature, pressure, component concentrations)
+   * pump efficiency of primary pumps and recycle pumps
+   * ERD pump efficiency and outlet pressure
+   * RO membrane solvent and solute permeability coefficients, feed-side channel height and spacer porosity, and permeate pressure
+   * LSR modules feed-side channel height and spacer porosity, and permeate pressure
+
+If configuring LSR stages to utilize fixed values of solvent and solute permeability coefficients:
+
+    * LSR membranes' solvent and solute permeability coefficients
+
+
+Flowsheet Specifications
+--------------------------
+
+.. csv-table::
+   :header: "Description", "Value", "Units"
+
+   "**Primary and booster pumps**", "", ""
+   "Pump efficiency", "0.75", "dimensionless"
+   "**ERDs**", "", ""
+   "efficiency", "0.8", "dimensionless"
+   "Outlet pressure", "101325", "Pa"
+   "**RO Stage**", "", ""
+   "Water permeability coefficient", "4.2E-12", "m/s-Pa"
+   "Salt permeability coefficient", "3.5E-8", "m/s"
+   "Feed-channel height", "1E-3", "m"
+   "Feed-side spacer porosity", "0.85", "dimensionless"
+   "Permeate pressure", "101325", "Pa"
+   "Minimum RO pressure", "10E5", "Pa"
+   "Maximum RO pressure", "85E5", "Pa"
+   "**LSRRO stages**", "", ""
+   "Water permeability coefficient", "2.78E-12 to 4.2E-11", "m/s-Pa"
+   "Salt permeability coefficient", "3.5E-8 to B_max", "m/s"
+   "Feed-channel height", "1E-3", "m"
+   "Feed-side spacer porosity", "0.85", "dimensionless"
+   "Minimum LSRRO pressure", "10E5", "Pa"
+   "Maximum LSRRO pressure", "65E5", "Pa"
+   "**Additional Specifications**", "", ""
+   "Permeate quality limit", "500E-6", "ppm"
+
+.. note::
+   When utilizing this model in research or publications, please cite the following reference:
+
+   Atia, A. A., Allen, J., Young, E., Knueven, B., & Bartholomew, T. V. (2023). Cost optimization of low-salt-rejection reverse osmosis. Desalination, 551, 116407.
```

Comment:
Move and/or copy this at the top of the document (@adam-a-a )

Reply:
done